packages/office-ui-fabric-react/src/components/Image/Image.scss

```diff
@@ -1,82 +1,80 @@
 @import '../../common/common';
 
-:global {
-  .ms-Image {
-    overflow: hidden;
-  }
+.root {
+  overflow: hidden;
+}
 
-  // Modifier: The image frame should fill its parent element.
-  .ms-Image--maximizeFrame {
-    height: 100%;
-    width: 100%;
-  }
+// Modifier: The image frame should fill its parent element.
+.maximizeFrame {
+  height: 100%;
+  width: 100%;
+}
 
-  .ms-Image-image {
-    display: block;
-    opacity: 0;
+.image {
+  display: block;
+  opacity: 0;
 
-    &.is-loaded {
-      opacity: 1;
-    }
+  &.isLoaded {
+    opacity: 1;
   }
+}
 
-  .ms-Image-image--center,
-  .ms-Image-image--contain,
-  .ms-Image-image--cover {
-    position: relative;
-    @include left(50%);
-    top: 50%;
-    // transform-translateX mixin is not used here because a transateY is also required.
-    @include LTR {
-      transform: translate(-50%,-50%);
-    }
-    @include RTL {
-      transform: translate(50%,-50%);
-    }
+.imageCenter,
```

Comment:
All of these modifiers should contain the root class they are modifying. E.g. `.rootIsImageCover`